doc/installationguide.md

```diff
@@ -206,9 +206,10 @@ used for the same purpose. For instance:
     any unexpected error.
 -   **singleConfigurationMode**: enables the Single Configuration mode for backwards compatibility (see description in
     the Overview). Default to false.
--   **timestamp**: if this flag is activated, the IoT Agent will add a 'TimeInstant' metadata attribute to all the
-    attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
+-   **timestamp**: if this flag is activated:
+    -   For NGSIv1/NGSIv2, the IoT Agent will add a `TimeInstant` metadata attribute to all the attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
     provision.
+    -   With NGSI-LD, the standard `observedAt` metadata attribute is created instead.
```

Comment:
```suggestion
    -   With NGSI-LD, the standard `observedAt` property-of-a-property is created instead.
```

As far as I understand, in NGSI-LD the concept of metadata doesn't exist natively and the right term is property-of-a-property (although I may be wrong).